mainwindow.cpp

```diff
@@ -862,7 +842,7 @@ void MainWindow::showJohnError(QProcess::ProcessError error)
 
 void MainWindow::showJohnFinished()
 {
-    // TODO: Should we place a message about it into output buffer?
+    appendLog(tr("Session ended."));
```

Comment:
Just append an equal "-----------------------" delimiter in same regard as the start of session.